packages/nocms-plugin-image-resources/test/createImageResourceProvider.spec.js

```diff
@@ -0,0 +1,94 @@
+import assert from 'assert';
+// import { createImageResourceProvider } from '../lib/createImageResourceProvider.js';
+import { activate } from '../lib/index.js';
+import fs from 'fs';
+
+const filePaths = [
+	'test/test-images/50mm-249.jpg',
+	'test/test-images/8-tallet.jpg'
+];
+
+let resourceProviders = [];
```

Comment:
Arrays can be consts - pushing to an array is not a reassign. Only a `resourceProviders = 'reassign` would be a reassign.

Reply:
I want to re-assign in the `beforeEach` for a fresh instance of the plugin (I think): 

```
	beforeEach(() => {
		resourceProviders = [];
```

Reply:
That makes sense, but my comment was also meant for how the code looks now :-D